### ISSUE TITLE
Pull to update filters

### ISSUE THREAD

**Issue by SpicySpaceman** [AdguardTeam/AdguardForiOS#645]
I think it could be quite usefull if you could just pull to update the filters like the mails in the mail app etc. instead of scrolling down to the update button every time. 

<details><summary>Screenshot:</summary>

![68fc2f4d-751c-4138-809c-a9e1b167d07f](https://user-images.githubusercontent.com/10995998/36049789-4a9a42ac-0de4-11e8-9a0b-2a6989dd6b07.jpeg)

</details>

### Your environment

* Device model and storage size: iPhone X
* Operating system and version: 11.3
* Browser: Safari
* AdGuard Pro 2.0.0 (132)

**Comment by worldsdream** [comment]
+1

**Comment by Goddess-Bastet** [comment]
+1.

**Comment by worldsdream** [comment]
Hi @Maxim-Butov will this be added in the next release or can you tell us more why it’s closed?

**Comment by danidoedel** [comment]
Yeah would be nice to know. Maybe you could give us a hint instead of closing it without any comment.

**Comment by iDjay** [comment]
+1

**Comment by planet0** [comment]
Still something that would be nice to have.
